### PR TITLE
I18n Sync In (with old structure) 06/03

### PR DESCRIPTION
**Pull request by Hamms**
This is a manual I18n sync with the pre-restructure structure, but with latest content. This will set us up to do the restructure sync later today without any content changes.

### CODE REVIEW

**Review by bethanyaconnor — APPROVED** [review]

**Comment by Hamms** [comment]
Note this also includes https://github.com/code-dot-org/code-dot-org/pull/28763, which adds several levels to dsls.en.yml

File: i18n/locales/source/dashboard/dsls.yml [hunk 1]
```diff
@@ -9,17 +9,384 @@ en:
       Begin planning your project_2018:
         Begin planning your project: Begin planning your project
         description here: description here
+      Begin planning your project_2018_2019:
+        Begin planning your project: Begin planning your project
+        description here: description here
+      BigEvent-Unplugged: {}
```
Comment (bethanyaconnor) [inline i18n/locales/source/dashboard/dsls.yml:15]: Why are there so many "empty" levels?

**Comment by Hamms** [comment]
Current Plan:

1. Sync up to crowdin the content in this PR
2. Sync up to crowdin the style-adjusted content in https://github.com/code-dot-org/code-dot-org/pull/28894
3. Manually upload to crowdin (in year order) the new course_content directory in https://github.com/code-dot-org/code-dot-org/pull/28903
4. Remove the relevant dashboard files from crowdin

**Comment by Hamms** [comment]
Update: We're going to wait for the levelbuilder scoop to catch some content fixes, then run steps 1-3, then manually check the course_content directory to verify that all duplicates were detected correctly. We expect there to be a few disparate strings, namely:

- Escaped backslashes in CSD content will be escaped differently
- Variables with underscores in Blockly XML will be escaped differently
- Emojis will be escaped differently

The first category is by far the largest, and we've decided we don't care strongly about losing those translations, since there are only a couple dozen of them and they're all in CSD. The other two categories have only a couple instances each and are in CSF, so we plan to fix those manually by applying machine translations.

**Review by nkiruka — APPROVED** [review]

**Review by bethanyaconnor — APPROVED** [review]

File: i18n/locales/source/dashboard/dsls.yml (hunk 1 above)
Comment (Hamms) [reply i18n/locales/source/dashboard/dsls.yml:15]: All the "empty" levels are external markdown, for which we don't yet capture their content. That feature is coming soon!

**Pull request merged**